Assets/UI/WorldView/citybannermanager.xml

```diff
@@ -158,23 +162,24 @@
     <Instance Name="OtherCityBanner">
         <ZoomAnchor ID="Anchor" ZoomOffsetNear="0,0,0" ZoomOffsetFar="0,0,-20">
             <Container Anchor="C,C" Offset="0,-20" Size="200,34">
-                <Button ID="CityBannerButton" Anchor="C,T" Size="Parent-20,34" NoStateChange="true">
-                    <Grid ID="CityBannerBackground" Anchor="C,T" Offset="1,0" Size="Parent,34" Color="255,255,255,150" SliceTextureSize="50,34" SliceCorner="25,17" Texture="Banner_BannerBacking">
-                        <Image ID="CityProductionProgress" Anchor="R,C" Offset="2,0" Size="14,32" TextureOffset="14,0" Speed="1" Texture="Banner_ProductionMeter">
-                            <TextureBar ID="CityProductionNextTurn" Anchor="L,C" Size="14,32" Direction="Up" Speed="1" Texture="Banner_ProductionMeter"/>
-                            <TextureBar ID="CityProductionMeter" Anchor="L,C" Size="14,32" Direction="Up" Speed="1" Texture="Banner_ProductionMeter"/>
+                <Button ID="CityBannerButton" Size="Parent-20,34" Anchor="C,T" NoStateChange="true">
```

Comment:
Any changes in this file beyond this point are just to match it to the unmodified version, specifically for the comparison of the files when using a file diffing tool.  None of the values were changed, just rearranged.  For example, line 172 is a change to the order of the XML attributes (putting Offset first, etc.) so the two files match.